frontends/common/resolveReferences/referenceMap.cpp

```diff
@@ -77,7 +80,8 @@ const IR::IDeclaration *ReferenceMap::getDeclaration(const IR::This *pointer, bo
 
 const IR::IDeclaration *ReferenceMap::getDeclaration(const IR::Path *path, bool notNull) const {
     CHECK_NULL(path);
-    auto result = get(pathToDeclaration, path);
+    auto it = pathToDeclaration.find(path);
+    auto result = it != pathToDeclaration.end() ? it->second : nullptr;
```

Comment:
We could and probably should add a `get` overload for the abseil maps to `lib/map.h`. These can be quite useful and especially for maps where the value type is a pointer but the map should never contain `nullptr` they make it much easier to use.

Reply:
Are we ok to have all abseil maps and corresponding includes to be exposed via `lib/map.h`?

Reply:
https://github.com/p4lang/p4c/pull/4483 generalizes `lib/map.h` to allow different maps.

Reply:
Thank you. I think it would make sense to merge #4483 first and reverse the `get` expansions in this PR.

Reply:
Yes, I will refine after #4483

Reply:
@vlstill This was refined, please take a look, thanks!